QualityControl/test/test-index.js

```diff
@@ -120,7 +122,12 @@ suite('All Tests - QCG', { timeout: FRONT_END_TIMEOUT + BACK_END_TIMEOUT }, asyn
       async (testParent) => await objectViewFromLayoutShowTests(url, page, FRONT_END_PER_TEST_TIMEOUT, testParent),
     );
 
-    // require('./layout-view.test');
+    test(
+      'should successfully run layout view page tests',
```

Comment:
It looks like there is a miss-match and confusing use of `view` and `show`. I see that the url parameter is `layoutShow`
Could you please adapt the name everywhere to use `show` instead of `view` when it comes to layout?